doc/source/whatsnew/v0.22.0.txt

```diff
@@ -195,5 +195,6 @@ Other
 ^^^^^
 
 - Improved error message when attempting to use a Python keyword as an identifier in a numexpr query (:issue:`18221`)
+- BLD: since we already use setuptools, let's remove the optional logic in setup.py (:issue:`18113`)
```

Comment:
This isn't the usual style of a what's new entry.  Something like "Simplified setup due to explicit dependence on setuptools (:issue...)"